iac-extensions/kubernetes/src/main/java/org/sonar/iac/kubernetes/visitors/LocationShifter.java

```diff
@@ -168,36 +166,18 @@ public static TextRange computeHelmValuePathTextRange(HelmInputFileContext helmC
     var goTemplateTree = helmContext.getGoTemplateTree();
     var sourceWithComments = helmContext.getSourceWithComments();
     if (goTemplateTree != null && sourceWithComments != null) {
-      try {
-        var contents = helmContext.inputFile.contents();
-        // The go template tree contains locations aligned to source code with additional trailing line numbers comments
-        var textRanges = GoTemplateAstHelper.findValuePathNodes(goTemplateTree, textRange, sourceWithComments)
-          .map(FieldNode::location)
-          .map(LocationShifter::fixLocation)
```

Comment:
Since locations are now more precise and shouldn't extend outside the text, I removed two workarounds we had: `fixLocation` and `trimEndToText`. Unit tests and ITs are passing, so it doesn't seem to have immediate drawbacks.